aquadoggo_cli/README.md

```diff
@@ -50,6 +50,21 @@ Options:
 
           eg. --rendezvous-peer-id "12D3KooWD3eckifWpRn9wQpMG9R9hX3sD158z7EqHWmweQAJU5SA"
 
+      --relay-client <RELAY_CLIENT>
```

Comment:
I intend on simplifying the CLI in a future PR (see https://github.com/p2panda/aquadoggo/issues/319).